src/modals/AddProjectModal.tsx

```diff
@@ -37,21 +37,15 @@ const AddProjectModal = () => {
     setColor(event.target.value);
   };
 
-  const handleNameChange = (event: {
-    target: { value: React.SetStateAction<string> };
-  }) => {
-    setName(event.target.value);
-  };
-
   const ProjectData = {
-    name,
+    nameRef,
```

Comment:
Api side expects an data with name. If you use nameRef instead of name, you will get an error.

What you can do is during the submitting,
name:nameRef

Reply:
Example

![image](https://user-images.githubusercontent.com/54946794/226286766-1694447a-1bcb-491a-917f-69d75600e2de.png)